docs/installation/web-interface.md

```diff
@@ -30,57 +30,29 @@ $ cd workspace
 $ podman run -p 8080:8080 -v "${PWD}:/workspace:z" --rm -it quay.io/konveyor/move2kube-aio:release-0.2
 ```
 
-**Latest** for development and testing cases:
+**Latest** (if you need bleeding edge features and also for development and testing use):
 
 To run locally using `docker`:
 
 ```shell
-$ mkdir -p workspace
-$ cd workspace
-$ docker run -p 8080:8080 -v "${PWD}:/workspace" -v /var/run/docker.sock:/var/run/docker.sock --rm -it quay.io/konveyor/move2kube-aio:latest
+$ docker run --rm -it -p 8080:8080 quay.io/konveyor/move2kube-ui:latest
 ```
+(Optional: If you need persistence then add `-v "${PWD}/move2kube-api-data:/move2kube-api/data"` to mount the current directory).  
```

Comment:
Could we possibly write the full command here? Not everyone might be well versed with where exactly to add that part. If they add in the end, it won't work.

Reply:
done